Disable host fragementation in open mode since hardware support it.

Signed-off-by: Zhu Yi <yi.zhu@intel.com>
---

 ipw2200-1.0.6-chuyee/ipw2200.c |    3 +++
 1 files changed, 3 insertions(+)

diff -L ipw2100.c -puN /dev/null /dev/null
diff -puN ipw2200.c~open_frag ipw2200.c
--- ipw2200-1.0.6/ipw2200.c~open_frag	2005-08-03 15:54:41.000000000 +0800
+++ ipw2200-1.0.6-chuyee/ipw2200.c	2005-08-03 15:56:55.000000000 +0800
@@ -8050,6 +8050,9 @@ static int ipw_sw_reset(struct ipw_priv 
 	}
 	IPW_DEBUG_INFO("Hardware crypto [%s]\n", hwcrypto ? "on" : "off");
 
+	/* IPW2200/2915 is abled to do hardware fragmentation. */
+	priv->ieee->host_open_frag = 0;
+
 	if ((priv->pci_dev->device == 0x4223) ||
 	    (priv->pci_dev->device == 0x4224)) {
 		if (init)
_
